sqlglot/parser.py

```diff
@@ -1129,6 +1138,26 @@ def _parse_with_property(
 
         return self._parse_withisolatedloading()
 
+    # https://dev.mysql.com/doc/refman/8.0/en/create-view.html
+    def _parse_definer(self) -> t.Optional[exp.Expression]:
+        self._match(TokenType.EQ)
+
+        user = self._parse_id_var()
+        self._match(TokenType.PARAMETER)
+        host = self._parse_id_var() or (self._match(TokenType.MOD) and self._prev.text)
+
+        if not user or not host:
+            return None
+
+        return exp.DefinerProperty(this=f"{user}@{host}")
+
+    def _parse_sqlsecurity(self) -> exp.Expression:
+        if self._match_text_seq("DEFINER"):
+            definer = True
+        elif self._match_text_seq("INVOKER"):
+            definer = False
```

Comment:
isn't defined undefined here? if it doesn't match these two branches?

Reply:
fixed